client/src/elm/Pages/PrenatalActivity/View.elm

```diff
@@ -1480,41 +1481,35 @@ viewNextStepsContent language currentDate isChw assembled data =
 
         viewTask task =
             let
-                ( iconClass, isCompleted ) =
+                iconClass =
                     case task of
                         NextStepsAppointmentConfirmation ->
-                            ( "next-steps-send-to-hc"
-                            , isJust measurements.appointmentConfirmation
-                            )
+                            "next-steps-send-to-hc"
 
                         NextStepsFollowUp ->
-                            ( "next-steps-follow-up"
-                            , isJust measurements.followUp
-                            )
+                            "next-steps-follow-up"
 
                         NextStepsSendToHC ->
-                            ( "next-steps-send-to-hc"
-                            , isJust measurements.sendToHC
-                            )
+                            "next-steps-send-to-hc"
 
                         NextStepsHealthEducation ->
-                            ( "next-steps-health-education"
-                            , isJust measurements.healthEducation
-                            )
+                            "next-steps-health-education"
 
                         NextStepsNewbornEnrolment ->
-                            ( "next-steps-newborn-enrolment"
-                            , isJust assembled.participant.newborn
-                            )
+                            "next-steps-newborn-enrolment"
 
                         NextStepsMedicationDistribution ->
-                            ( "next-steps-medication-distribution"
-                            , isJust measurements.medicationDistribution
-                            )
+                            "next-steps-treatment"
```

Comment:
@anvmn Please confirm this next step is correct, it doesn't match with the previous one.

Reply:
@mariano-dagostino 

This is correct. What we define here is the icon that will be presented for activity, and these 2 share same icon.